frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -82,10 +81,16 @@ const FillRemainingSpace = styled.span`
   flex: 1 1 auto;
 `;
 
+export interface ISearchFields {
```

Comment:
A empty string is still of type `string`. Any reason for this?

Reply:
yeah, these are redundant, will fix.